builder/Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM ubuntu
```

Comment:
Does this have to be ubuntu? If so you should at least pin to a specific version e.g 22_04 or something other than just "latest".